prisma/migrations/20240706000412_one_day_referrals/migration.sql

```diff
@@ -0,0 +1,33 @@
+-- CreateEnum
+CREATE TYPE "OneDayReferralType" AS ENUM ('REFERRAL', 'ITEM', 'COMMENT', 'PROFILE', 'TERRITORY');
```

Comment:
I had them that way at one point but changed it because I couldn't determine which is which from `/items/<id>` style links. Maybe I don't need to be able to tell on the links though hmmm.

Reply:
Ah, I see